backend/docker-compose.test-backend.yml

```diff
@@ -17,8 +17,6 @@ services:
             BACKEND_HOST: ${SUPERVISOR_HOST}
             SUPERVISOR_HOST: ${SUPERVISOR_HOST}
             SUPERVISOR_PORT: ${SUPERVISOR_PORT}
-        volumes:
-            - ./testing/configs/:/root/configs/
```

Comment:
I think that messages.json should at least be mapped via a volume mapping so that we can change them without having to rebuild. What do you think?